src/main/java/hudson/plugins/accurev/delegates/StreamDelegate.java

```diff
@@ -88,6 +91,31 @@ protected PollingResult checkForChanges(Job<?, ?> project)
           .println("Tried to find '" + localStream + "' Stream, could not found it.");
       return PollingResult.NO_CHANGES;
     }
+    // command to get version of accurev
+    final ArgumentListBuilder cmd = new ArgumentListBuilder();
+    cmd.add("accurev");
+    String ACCUREV_VERSION =
+        AccurevLauncher.runJustAccurev(
```

Comment:
I doubt you need to create a separate method for the Launcher.

You should be able to call `AccurevLauncher.runCommand()` with an empty argumentList `final ArgumentListBuilder cmd = new ArgumentListBuilder()`

Reply:
Ok i will try that i push changes asap. Got to get merged today